src/auditlogs.js

```diff
@@ -55,7 +55,7 @@ class FirehoseLog extends events.EventEmitter {
   }
 
   _scheduleFlush() {
-    if (!this._flushTImer) {
+    if (!this._flushTimer) {
```

Comment:
This was likely also causing more flushes than necessary, right?  In fact, one per log entry..